product_docs/docs/pem/8/considerations/authentication_options/configuring_the_pem_server_to_use_windows_kerberos_server.mdx

```diff
@@ -170,19 +170,19 @@ It will display the principal along with the kerberos ticket.
 
 -   Run the PEM configure script on the PEM Server to use Kerberos Authentication
 
-    ```text
+    ```ini
```

Comment:
@ccestes This one is a script/command that you run in the terminal. It should be shell not ini.